engines/order_management/app/services/order_management/subscriptions/proxy_order_syncer.rb

```diff
@@ -90,8 +90,7 @@ def not_closed_in_range_order_cycles
       end
 
       def in_range_order_cycles
-        ends_at_or_limit = ends_at || 100.years.from_now
-        order_cycles.where(orders_close_at: begins_at..ends_at_or_limit )
+        order_cycles.where(orders_close_at: begins_at..ends_at)
```

Comment:
💯 
I was wondering this in the previous commit. I'm glad I looked ahead before making a suggestion!